.gitignore

```diff
@@ -108,14 +102,15 @@ CMakeDoxy*
 compile_commands.json
 
 ## Android build files
-build/android/src/main/assets
 build/android/build
-build/android/deps
-build/android/libs
-build/android/jni/lib
-build/android/jni/src
-build/android/src/main/jniLibs
-build/android/obj
 build/android/local.properties
 build/android/.gradle
+build/android/app/build
+build/android/app/release
+build/android/app/src/main/assets
+build/android/native/.*
+build/android/native/build
+build/android/native/deps
 timestamp
+.idea
```

Comment:
A: this line is duplicate of L24
B: I vote for a new `build/android/.gitignore` and move all `build/android/xxxxx` entries to this file to have complexity to the place where it belongs
C: Instead of `.idea` I recommend to add
```
.idea/*
!.idea/codeStyles/
```
to have only codestyle checked in

Reply:
Well noticed, I'll do it.